### PR TITLE
[ASTextNode] Fixes in ASTextKitFontSizeAdjuster

### PR DESCRIPTION
**Pull request by ejensen**
- Removes double scaling of `paragraphSpacing`.
- Removes scaling of `lineHeightMultiple` attribute since it is a scale of the `lineHeight` which is already being adjusted.
- Adds Unit tests and Screenshot tests for `ASTextKitFontSizeAdjuster`.

diff

### CODE REVIEW

File: Source/TextKit/ASTextKitFontSizeAdjuster.mm [hunk 1]
```diff
@@ -87,8 +87,6 @@ + (void)adjustFontSizeForAttributeString:(NSMutableAttributedString *)attrString
       paragraphStyle.tailIndent = (paragraphStyle.tailIndent * scaleFactor);
       paragraphStyle.minimumLineHeight = (paragraphStyle.minimumLineHeight * scaleFactor);
       paragraphStyle.maximumLineHeight = (paragraphStyle.maximumLineHeight * scaleFactor);
-      paragraphStyle.lineHeightMultiple = (paragraphStyle.lineHeightMultiple * scaleFactor);
-      paragraphStyle.paragraphSpacing = (paragraphStyle.paragraphSpacing * scaleFactor);
```
Comment (ejensen) [inline Source/TextKit/ASTextKitFontSizeAdjuster.mm]: `paragraphSpacing` is already being scaled in line 84. Repeating the scaling cases the paragraph spacing mismatch the font point scaling.

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2218,36 +2219,6 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
-		3B9D88CDF51B429C8409E4B6 /* [CP] Copy Pods Resources */ = {
```
Comment (ejensen) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: I could not run the tests due to the Cocoapods build script throwing an error. The error message recommended running `pod install`. The resulting change was the removal of these two run script phases. With them gone, I can now run the tests.

Comment (garrettmoon) [reply AsyncDisplayKit.xcodeproj/project.pbxproj]: Could this be due to differing versions of cocoa pods?

Comment (ejensen) [reply AsyncDisplayKit.xcodeproj/project.pbxproj]: I'm not sure what caused it. I have rebased my code changes onto `master` and the issue went away. So I've reverted this project file change.

**Pull request merged**